esphome/components/esphome/ota/__init__.py

```diff
@@ -0,0 +1,152 @@
+from esphome.cpp_generator import RawExpression
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome import automation
+from esphome.const import (
+    CONF_ID,
+    CONF_NUM_ATTEMPTS,
+    CONF_OTA,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_REBOOT_TIMEOUT,
+    CONF_SAFE_MODE,
+    CONF_TRIGGER_ID,
+    CONF_VERSION,
+    KEY_PAST_SAFE_MODE,
+)
+from esphome.core import CORE, coroutine_with_priority
+
+
+CODEOWNERS = ["@esphome/core"]
+AUTO_LOAD = ["md5", "socket"]
+DEPENDENCIES = ["network", "ota"]
```

Comment:
Not required as this is an ota platform so it will always be loaded
```suggestion
DEPENDENCIES = ["network"]
```

Reply:
🤦🏻‍♂️